system-tests/tests/smoke/cre/por_test.go

```diff
@@ -49,11 +52,12 @@ import (
 	creworkflow "github.com/smartcontractkit/chainlink/system-tests/lib/cre/workflow"
 	libcrecli "github.com/smartcontractkit/chainlink/system-tests/lib/crecli"
 	keystoneporcrecli "github.com/smartcontractkit/chainlink/system-tests/lib/crecli/por"
+	libfunding "github.com/smartcontractkit/chainlink/system-tests/lib/funding"
 	libtypes "github.com/smartcontractkit/chainlink/system-tests/lib/types"
 )
 
 var (
-	SinglePoRDonCapabilitiesFlags = []string{"ocr3", "cron", "custom-compute", "write-evm"}
+	SinglePoRDonCapabilitiesFlags = []string{"ocr3", "cron", "custom-compute", "write-evm", "read-contract"}
```

Comment:
I am a bit on the fence when it comes to whether we should use the same set of flags for both tests. On the one hand if the DON has a capacity that is not used by any workflow it should work. It should also work, if the binary is missing, but not used by any workflow.

On the other hand that will result in errors, which will obscure the logs.

What's your take on ti?